lib/handlers/handler.ts

```diff
@@ -108,114 +108,129 @@ export abstract class APIGLambdaHandler<CInj, RInj extends BaseRInj, ReqBody, Re
     }
   }
 
+  protected async processRequestInEntirety(
```

Comment:
this feels dangerous, this function could be overriden by somebody else without calling super